models/sessions_test.go

```diff
@@ -18,43 +20,63 @@ type SessionsTestSuite struct {
 func (ts *SessionsTestSuite) SetupTest() {
 	TruncateAll(ts.db)
 	email := "test@example.com"
-	user, err := NewUser("", email, "secret", "test", nil)
+	user, err := NewUser("", email, "secret", ts.Config.JWT.Aud, nil)
 	require.NoError(ts.T(), err)
 
 	err = ts.db.Create(user)
 	require.NoError(ts.T(), err)
 }
 
+func TestSession(t *testing.T) {
+	globalConfig, err := conf.LoadGlobal(modelsTestConfig)
+	require.NoError(t, err)
+	conn, err := test.SetupDBConnection(globalConfig)
+	require.NoError(t, err)
+	ts := &SessionsTestSuite{
+		db:     conn,
+		Config: globalConfig,
+	}
+	defer ts.db.Close()
+	if globalConfig.MFA.Enabled {
+		suite.Run(t, ts)
+	}
+}
+
 func (ts *SessionsTestSuite) TestCalculateAALAndAMR() {
 	totalDistinctClaims := 2
 	u, err := FindUserByEmailAndAudience(ts.db, "test@example.com", ts.Config.JWT.Aud)
 	require.NoError(ts.T(), err)
-	session, err := CreateSession(ts.db, u, &uuid.Nil)
+	// TODO(Joel): Replace with CreateSession once MFA feature flag is lifted
+	session, err := MFA_CreateSession(ts.db, u, &uuid.Nil)
 	require.NoError(ts.T(), err)
 
 	err = AddClaimToSession(ts.db, session, PasswordGrant)
 	require.NoError(ts.T(), err)
 
-	firstClaimAddedTime := time.Now().Unix()
+	firstClaimAddedTime := time.Now()
```

Comment:
just checking - did we decide on returning unix time here?

Reply:
Don't think so but the APIs do return Unix time. This is tested with `.Before()` and `time` as the granularity of unix timestamp is a second and the time difference between the first claim being added and the second claim being added is less than a second. As such, the timestamp update on the claim is not discrenible when using Unixtime.

One alternative considered here is to use a `.sleep` function for a second -- this works but would slow down the test.

Open to other suggestions as well